docs/features/software-templates/writing-custom-actions.md

```diff
@@ -305,78 +300,6 @@ If you'll preserve the same key, and you'll try to restart the affected task, it
-| Utility actions          | [scaffolder-backend-module-utils](https://www.npmjs.com/package/@roadiehq/scaffolder-backend-module-utils)                                      | [Roadie](https://roadie.io)                                  |
-| AWS cli actions          | [scaffolder-backend-module-aws](https://www.npmjs.com/package/@roadiehq/scaffolder-backend-module-aws)                                          | [Roadie](https://roadie.io)                                  |
-| Scaffolder .NET Actions  | [plugin-scaffolder-dotnet-backend](https://www.npmjs.com/package/@plusultra/plugin-scaffolder-dotnet-backend)                                   | [Alef Carlos](https://github.com/alefcarlos)                 |
-| Scaffolder Git Actions   | [plugin-scaffolder-git-actions](https://www.npmjs.com/package/@mdude2314/backstage-plugin-scaffolder-git-actions)                               | [Drew Hill](https://github.com/arhill05)                     |
-| Azure Pipeline Actions   | [scaffolder-backend-module-azure-pipelines](https://www.npmjs.com/package/@parfuemerie-douglas/scaffolder-backend-module-azure-pipelines)       | [Parfümerie Douglas](https://github.com/Parfuemerie-Douglas) |
-| Azure Repository Actions | [scaffolder-backend-module-azure-repositories](https://www.npmjs.com/package/@parfuemerie-douglas/scaffolder-backend-module-azure-repositories) | [Parfümerie Douglas](https://github.com/Parfuemerie-Douglas) |
-| Snyk Import Project      | [plugin-scaffolder-backend-module-snyk](https://www.npmjs.com/package/@ma11hewthomas/plugin-scaffolder-backend-module-snyk)                     | [Matthew Thomas](https://github.com/Ma11hewThomas)           |
-| JSON Merge Actions       | [plugin-scaffolder-json-merge-actions](https://www.npmjs.com/package/@mdude2314/backstage-plugin-scaffolder-json-merge-actions)                 | [Drew Hill](https://github.com/arhill05)                     |
-| NPM Actions              | [plugin-scaffolder-npm-actions](https://www.npmjs.com/package/@mdude2314/backstage-plugin-scaffolder-npm-actions)                               | [Drew Hill](https://github.com/arhill05)                     |
-| Slack Actions            | [plugin-scaffolder-backend-module-slack](https://www.npmjs.com/package/@mdude2314/backstage-plugin-scaffolder-backend-module-slack)             | [Drew Hill](https://github.com/arhill05)                     |
-| Microsoft Teams Actions  | [plugin-scaffolder-backend-module-ms-teams](https://www.npmjs.com/package/@grvpandey11/backstage-plugin-scaffolder-backend-module-ms-teams)     | [Gaurav Pandey](https://github.com/grvpandey11)              |
-
-Have fun! 🚀
+You can find a list of community-contributed actions here in our [contrib](https://github.com/backstage/backstage/blob/master/contrib/scaffolder/custom-action-packages.md) docs!
```

Comment:
How do we feel about this? I have created and moved the community/OS ones to a file in contrib, I think this is where we stand on adding these to the docs right? as where does the line get drawn 😅

Reply:
![image](https://github.com/user-attachments/assets/13c72134-83d3-4d3d-aaee-2af4799d5a66)

Reply:
Ohh! Do you mean like https://backstage.io/plugins then filtering on scaffolder? that would be nice! I just tried that though and I don't think the filter gets passed to the url params 😞

Reply:
Yeah, that's the idea and there are no query params which is what I was wishing there was.

Reply:
Alternatively I could always just document to do that 😅 

"Go to https://backstage.io/plugins and filter on scaffolder"

WDYT?

Reply:
That feels a bit more maintained and less hidden even if the user experience isn't the best. IMO at least.

Reply:
[0592917](https://github.com/backstage/backstage/pull/29612/commits/0592917347354e83059a52101d7ae1a83b6821e3) 🙏